wordpress-varnish-as-a-service.php

```diff
@@ -55,6 +55,10 @@ public function __construct() {
 		add_action('add_attachment', array(&$this, 'WPVarnishPurgeAttachment'), 99);
 		add_action('edit_attachment', array(&$this, 'WPVarnishPurgeAttachment'), 99);
 		add_action('delete_attachment', array(&$this, 'WPVarnishPurgeAttachment'), 99);
+		add_action('wp_update_nav_menu', array(&$this, 'WPVarnishPurgeAll'), 99);
```

Comment:
These menu action hooks have already been approved + merged into rc, correct?

Reply:
They were approved and merged in. Not sure why it's still showing up here. I'll see if it cause conflict when I merge in.